src/java/org/apache/cassandra/io/sstable/format/PartitionIndexIterator.java

```diff
@@ -34,6 +34,11 @@
      */
```

Comment:
The class's JavaDoc is no longer true -- `isExhausted` should be used instead.

Reply:
ok

Reply:
I suppose we should rename it? Maybe just `PartitionKeyIterator` would be more suitable for the interface name?